packages/code-studio/src/main/AppMainContainer.tsx

```diff
@@ -957,6 +952,7 @@ export class AppMainContainer extends Component<
           }
           subtitle="Please check your network connection."
         />
+        {/* TODO: Reconnect auth fail - part of the plugin?? AppBootstrap? */}
```

Comment:
Ticket for this?

Reply:
Filed #1256, and I'll just remove the comment from here. It was a note for later, and an omission when implementing #1149.